### PR TITLE
Use of label field in animations example

### PR DESCRIPTION
**Pull request by yash1200**
In this PR, I have updated the `BottomNavigationBarItem` to use `label` field instead of deprecated `title` field in `/packages/animations/example/lib/fade_through_transition.dart` file.

### CODE REVIEW

File: packages/palette_generator/example/ios/Flutter/flutter_export_environment.sh [hunk 1]
```diff
@@ -7,4 +7,4 @@ export "FLUTTER_BUILD_DIR=build"
 export "SYMROOT=${SOURCE_ROOT}/../build/ios"
 export "FLUTTER_FRAMEWORK_DIR=/Users/gspencer/code/flutter/bin/cache/artifacts/engine/ios"
 export "FLUTTER_BUILD_NAME=0.1.0"
-export "FLUTTER_BUILD_NUMBER=0.1.0"
+export "FLUTTER_BUILD_NUMBER=0.1.0"
```
Comment (goderbauer) [inline packages/palette_generator/example/ios/Flutter/flutter_export_environment.sh:10]: Can you revert this unrelated change?

**Review by goderbauer — APPROVED** [review]
LGTM after nit is resolved.

File: packages/palette_generator/example/ios/Flutter/flutter_export_environment.sh (hunk 1 above)
Comment (yash1200) [reply packages/palette_generator/example/ios/Flutter/flutter_export_environment.sh]: Yep, it is done.

**Review by goderbauer — APPROVED** [review]
LGTM
